gotocompany/optimus/core/v1beta1/job_spec.proto

```diff
@@ -287,6 +287,7 @@ message ChangeJobNamespaceResponse {}
 message ListJobSpecificationRequest {
   string project_name = 1;
   string namespace_name = 2;
+  bool disable_asset = 3;
```

Comment:
`disable` seems off and also let's use the plural word. 
how about `exclude_assets` or `ignore_assets`?

Reply:
sure, it's better, let's use `ignore_assets` one